DataCollection/src/Domain/Entities/SurveyCompleted.cs

```diff
@@ -16,10 +18,10 @@ public class SurveyCompleted : IEntity
         public int LocationId { get; set; }
 
         [Required]
-        public DateTime StartDate { get; set; }
+        public DateTime? StartDate { get; set; }
```

Comment:
Had to change to nullable in order to satisfy the new interface. However, the field is still required. Which reminds me, I have not built-in any validation before converting a pending survey to a completed one.